File: easy/add-digits/add-digits.js
var addDigits = function(num) {
    let arrayOfDigits = Array.from(String(num), Number);
    while(arrayOfDigits.length !=1){
        suma = arrayOfDigits.reduce((a,b) => a+b);
        arrayOfDigits = Array.from(String(suma),Number);
    }
    return arrayOfDigits[0]
};

File: easy/add-digits/add-digits.py
def addDigits(self, num):
        """
        :type num: int
        :rtype: int
        """
        listdig = [int(x) for x in str(num)]
        while len(listdig)!=1:
            suma = sum(listdig)
            listdig= [int(x) for x in str(suma)]
        return listdig[0]